Multisig/UI/App/UIKit/UI Library/ReusableViews/AddressInfoView.swift

```diff
@@ -8,18 +8,29 @@
 
 import UIKit
 
-class AddressWithTitleView: UINibView {
+struct AddressInfo {
+    let address: Address
+    let label: String?
+}
+
+class AddressInfoView: UINibView {
     @IBOutlet private weak var identiconView: UIImageView!
     @IBOutlet private weak var textLabel: UILabel!
     @IBOutlet private weak var detailLabel: UILabel!
-    @IBOutlet private weak var button: UIButton!
+    @IBOutlet private weak var copyButton: UIButton!
+    @IBOutlet private weak var disclosureButton: UIButton!
+
+    private (set) var addressInfo: AddressInfo!
+
+    var onAddressInfoSelection: (() -> Void)?
+    var onDisclosureButtonAction: (() -> Void)?
 
-    private var style: Style = .nameAndAddress
-    private (set) var address: Address!
+    @IBAction private func onDisclosureButton() {
```

Comment:
In our code base the code layout is usually this:

```
 public vars
 private vars
 init
public methods
private methods
```

Can you please move these function below init or at the end somewhere?
Also, public vars would be better in the beginning of the class

Reply:
Ok